cli/utils/url.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2016 IBM Corporation
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package utils
+
+import (
+	"net/url"
+)
+
+// IsURL returns true if the given string is a URL
+func IsURL(rawURL string) bool {
```

Comment:
Consider dropping this. Wrapping ParseRequestURI (or just url.Parse) when it is used only twice seems like overhead to me (considering that it comes with its own test file and used only twice in non test code).